ports/mimxrt10xx/supervisor/port.c

```diff
@@ -410,11 +417,15 @@ void port_idle_until_interrupt(void) {
         NVIC_ClearPendingIRQ(SNVS_HP_WRAPPER_IRQn);
         // Don't down clock on debug builds because it prevents the DAP from
         // reading memory
+        #if 0
         #if CIRCUITPY_DEBUG == 0
         CLOCK_SetMode(kCLOCK_ModeWait);
         #endif
         __WFI();
         CLOCK_SetMode(kCLOCK_ModeRun);
+        #else
```

Comment:
Don't `#if 0` here. You can just remove the bad code. You can include it commented out in the comment above, but we can also always use git to find the old code.